test/fixtures/duplicates.input.js

```diff
@@ -0,0 +1,2 @@
+a.a = 1;
+a.a = 1;
```

Comment:
Out of curiousity, will this change eliminate this case?

    a.a = 1;
    a.a = 2;

...and what about this one?

    a.a = 1;
    var foo = a.a;
    a.a = 2;
    doSomethingWith(foo);